src/metadata/metadata_handler.h

```diff
@@ -46,20 +46,23 @@ class CdsObject;
 class IOHandler;
 
 // content handler Id's
-#define CH_DEFAULT 0
-#define CH_LIBEXIF 1
-#define CH_ID3 2
-#define CH_TRANSCODE 3
-#define CH_EXTURL 4
-#define CH_MP4 5
-#define CH_FFTH 6
-#define CH_FLAC 7
-#define CH_FANART 8
-#define CH_MATROSKA 9
-#define CH_SUBTITLE 10
-#define CH_CONTAINERART 11
-#define CH_WAVPACK 12
-#define CH_RESOURCE 20
+enum class ContentHandler {
+    UNKNOWN = -1,
```

Comment:
Why do we need a UNKNOWN type? Wouldn't DEFAULT work better? and then it can be unsigned too.